pkg/validation/validate_cluster.go

```diff
@@ -18,56 +18,50 @@ package validation
 
 import (
 	"fmt"
-	"net/url"
-	"time"
-
 	"net"
+	"net/url"
 
+	"github.com/golang/glog"
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/kops/pkg/apis/kops"
 	"k8s.io/kops/pkg/apis/kops/util"
+	"k8s.io/kops/pkg/cloudinstances"
+	"k8s.io/kops/pkg/dns"
+	"k8s.io/kops/upup/pkg/fi/cloudup"
 )
 
 // ValidationCluster a cluster to validate.
 type ValidationCluster struct {
```

Comment:
So this is a breaking change. I know people are using the yaml and json progrmatically. I wanted to have an api for this, but I think we could not decide on it. Options?

Reply:
Let me ask a question. Does this change the validation output?